spk/emby/src/installer.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+
+# Package
+PACKAGE="emby"
+DNAME="Emby"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+INSTALL_LOG="${INSTALL_DIR}/var/install.log"
+TMP_INSTALL_LOG="${TMP_DIR}/${PACKAGE}/var/install.log"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+USER="${PACKAGE}"
+GROUP="users"
+
+SERVICETOOL="/usr/syno/bin/servicetool"
+FWPORTS="/var/packages/${PACKAGE}/scripts/${PACKAGE}.sc"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Logging Install
+    mkdir -p ${INSTALL_DIR}/var
+    echo "|| Installing package $(grep "version" /var/packages/${PACKAGE}/INFO) - $(date) ||" >> ${INSTALL_LOG}
+
+    # Install busybox stuff
+    ${INSTALL_DIR}/bin/busybox --install ${INSTALL_DIR}/bin
+
+    # Create user
+    adduser -h ${INSTALL_DIR}/var -g "${DNAME} User" -G ${GROUP} -s /bin/sh -S -D ${USER}
+
+    # Correct the files ownership
+    chown -R ${USER}:root ${SYNOPKG_PKGDEST}
+
+    # Add firewall config
+    ${SERVICETOOL} --install-configure-file --package ${FWPORTS} >> /dev/null
+
+    # Log
+    echo "|| Package Install Completed - $(date) ||" >> ${INSTALL_LOG}
+
+    exit 0
+}
+
+preuninst ()
+{
+    # Stop the package
+    ${SSS} stop > /dev/null
+
+    # Remove the user (if not upgrading)
+    if [ "${SYNOPKG_PKG_STATUS}" != "UPGRADE" ]; then
+        delgroup ${USER} ${GROUP}
+        deluser ${USER}
+    fi
+
+    # Remove firewall config
+    if [ "${SYNOPKG_PKG_STATUS}" == "UNINSTALL" ]; then
+        ${SERVICETOOL} --remove-configure-file --package ${PACKAGE}.sc >> /dev/null
+    fi
+
+    exit 0
+}
+
+postuninst ()
+{
+    # Remove Link
+    rm -f ${INSTALL_DIR}
+
+    exit 0
+}
+
+preupgrade ()
+{
+    # Log Upgrade
+    echo "|| Beginning Package Upgrade - $(date) ||" >> ${INSTALL_LOG}
```

Comment:
You might have reused Sonarr's code? I know that logging (initially) was interesting for Sonarr, but does Emby have the same level of 'possible issues' that an upgrade should be logged like this? It's not all that likely we'll be doing anything with the info, because it's not all that informative tbh ;)
Fwiw, I'm fine with leaving it in during the beta, but outside of that, I'd propose to remove it.